curriculum/challenges/english/21-a2-english-for-developers/learn-greetings-in-your-first-day-at-the-office/pleasure-to-meet-you.md

```diff
@@ -0,0 +1,49 @@
+---
+id: 65528a7c39eb38f51e0e6f7f
+title: "Pleasure to Meet You"
```

Comment:
Nice catch. Shoud I click on the Commit suggestion button or is it better another way?

Reply:
You can click the commit suggestion button - that will make two commits, one for each suggestion. Otherwise, typically, we like to click on the "files" tab at the top, find the two suggestions and click "add suggestion to batch" next to each of them - then once you've batched all the ones you want to add (those two), click the commit button, and it will only add one commit.